components/post-text/PostText.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Link from 'next/link';
-import { uniq as _uniq } from 'lodash';
+import _uniq from 'lodash/uniq';
```

Comment:
nice improvement to reduce package size

Reply:
I also changed the issue title to check all loadash import in order to reduce package size: https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/issues/29